recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -90,3 +90,69 @@ add_task(withSandboxManager(Assert, async function testCreateStorage(sandboxMana
     })();
   `);
 }));
+
+add_task(withDriver(Assert, async function addons(driver) {
+  const ADDON_ID = "normandydriver@example.com";
+  const startDate = new Date();
+  const dir = getChromeDir(getResolvedURI(gTestPath));
+  dir.append("fixtures");
+  dir.append("normandy.xpi");
+  const xpiUrl = Services.io.newFileURI(dir).spec;
+
+  let addon = await driver.addons.get(ADDON_ID);
+  Assert.equal(addon, null, "Add-on is not yet installed");
+
+  await driver.addons.install(xpiUrl);
+  addon = await driver.addons.get(ADDON_ID);
+
+  Assert.notEqual(addon, null, "Add-on object was returned");
+  // compare with seconds-level accuracy
```

Comment:
I'm not a huge fan of depending on the timing of this; maybe we should just assert the type of the date over asserting it's recentness?